### ISSUE TITLE
Dataloder questions about "Continual Few-shot Relation Learning via Embedding Space Regularization and Data Augmentation"

### ISSUE THREAD

**Issue by xjtuleeyf** [qcwthu/Continual_Fewshot_Relation_Learning#8]
Hi, Chengwei, thanks for your excellent work!
In your work, the data was divided into three parts:training_data, valid_data and test_data. 
1.The first question is that why the correct label is included in the candidate options in the valid_data, but not in the training_data? As shown in the figure below：Is there any difference between them?
2.Besides, I notioced that the data in val_data and test_data are completely same. In my opinion, the function of valid_data is to fine-tune the parameters of the model as the sample source of query set in the training phase, so I studied the training code, and found that validation set data was not used in the training phase. So how do you use the valid_data? 
3.I found that there is a small overlap between the data used for training and the valid_data(test_data), is there any special reason for that? 
Finally, I want to experiment with my new model based on your training paradigm on the basis of your research, so I want to make sure whether my understanding of your training paradigm is correct: 
Take fewrel dataset as an example, in the basic training stage of k=0, select 10 relationships, 100 for each relationship as training samples, and in the incremental learning stage (k>0), a group of 10 relationships, Select 2-5-10 samples as training data for each relationship according to different tasks(2-shot,5-shot,10-shot), and then select 1-2 samples as valid data different with training data to fine-tuning model for each relationship, finally select test samples that are different from the training and vilid to test the performance of the model (test data are only selected from the relationships seen).
I haven't studied this field for a long time, so I really hope you can point out some mistakes in the expressions.
I would be appreciate if you could answer me, that will help me a lot. Thanks!
